src/components/App.tsx

```diff
@@ -18,21 +18,23 @@ import { hot } from 'react-hot-loader/root';
 import { observer } from 'mobx-react-lite';
 import * as React from 'react';
 import { ToastProvider } from 'react-toast-notifications';
+import { useMemo } from 'react';
 import StoresProvider from './StoresProvider';
 import Toast from './notifications/Toast';
 import ToastContainer from './notifications/ToastContainer';
 import Notifier from './notifications/Notifier';
 import WorkspacesLayout from './workspace/WorkspacesLayout';
 import Graph from './graph/Graph';
-import WorkspaceLinkGetter from './WorkspaceLinkGetter';
 import MessagesViewConfigurator from './messages-view-configurator/MessagesViewConfigurator';
-import { useSearchStore } from '../hooks/useSearchStore';
-import { useTabsStore } from '../hooks';
+import { useSessionsStore } from '../hooks';
 import '../styles/root.scss';
 
 const AppRootBase = () => {
-	const searchStore = useSearchStore();
-	const tabsStore = useTabsStore();
+	const sessionsStore = useSessionsStore();
+
+	const sessionNames = useMemo(() => sessionsStore.sessions.map(({ session }) => session), [
```

Comment:
even though that's ok to use useMemo here, but wouldn't `computed` make more sense, since we are deriving it from observable array ?

Reply:
fixed: https://github.com/th2-net/th2-rpt-viewer/pull/322/commits/fe67f8dcf54fe187157b865a90648d7ea6df6574